mailbox/store/src/main/java/org/apache/james/mailbox/store/StoreRightManager.java

```diff
@@ -138,16 +139,14 @@ public void applyRightsCommand(MailboxPath mailboxPath, ACLCommand mailboxACLCom
         assertSharesBelongsToUserDomain(mailboxPath.getUser(), mailboxACLCommand);
         MailboxMapper mapper = mailboxSessionMapperFactory.getMailboxMapper(session);
         Mailbox mailbox = mapper.findMailboxByPathBlocking(mailboxPath);
-        ACLDiff aclDiff = mapper.updateACL(mailbox, mailboxACLCommand);
-
-        eventBus.dispatch(EventFactory.aclUpdated()
-            .randomEventId()
-            .mailboxSession(session)
-            .mailbox(mailbox)
-            .aclDiff(aclDiff)
-            .build(),
-            new MailboxIdRegistrationKey(mailbox.getMailboxId()))
-            .block();
+        mapper.updateACL(mailbox, mailboxACLCommand)
```

Comment:
this looks pretty much the same what `setRights()` is doing

Reply:
except it uses `setACL`